src/cli.ts

```diff
@@ -135,7 +136,15 @@ async function writeAndImport(
   filepath: string,
   origin = filepath
 ) {
-  await fs.writeFile(filepath, script.toString())
+  const contents = script.toString()
+  await fs.writeFile(filepath, contents)
+
+  if (argv.import) {
+    const deps =
+      argv.import === true ? parseDeps(contents) : parseImportFlag(argv.import)
```

Comment:
Let’s simplify this flag. And allow it to be only on/off. 

Versions of packages lets parse from script content.